jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -130,38 +130,43 @@ public interface Request extends Attributes, Content.Source
     String getId();
 
     /**
-     * @return the {@link Components} to be used with this request.
+     * @return the {@link Components} to be used with this {@code Request}.
      */
     Components getComponents();
 
     /**
-     * @return the {@code ConnectionMetaData} associated to this request
+     * @return the {@code ConnectionMetaData} associated to this {@code Request}
      */
     ConnectionMetaData getConnectionMetaData();
 
     /**
-     * @return the HTTP method of this request
+     * @return the HTTP method of this {@code Request}
      */
     String getMethod();
 
     /**
-     * @return the HTTP URI of this request
+     * @return the HTTP URI of this {@code Request}
      * @see #getContextPath(Request)
      * @see #getPathInContext(Request)
      */
     HttpURI getHttpURI();
 
     /**
+     * Get the {@link Context} associated with this {@code Request}.
+     * If the request is not being handled by a {@link org.eclipse.jetty.server.handler.ContextHandler} then
+     * the {@link org.eclipse.jetty.server.Server.ServerContext} instance will be returned.
```

Comment:
Again seems overspecified, I would remove this, also because it is already explained in the `Context` javadocs.

Reply:
I think it needs to explain that this is never null.

Reply:
I agree it should say it is never null, but I don't agree to specify what subclass is returned.